<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogoGenerarBotones</class>
 <widget class="QDialog" name="DialogoGenerarBotones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialogo Generar Botones</string>
  </property>
  <widget class="QLineEdit" name="lineEditBotones">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonGenerer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Generar Botones</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInfoBoton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCambiar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cambiar Texto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
